src/components/About/About.scss

```diff
@@ -1,14 +1,12 @@
-.root {
+.rootAbout {
```

Comment:
Use inheritance in scss. For example, if div with `className = textAbout` is in div with `className = rootAbout`, then you should write it as:
```
.rootAbout {
  .textAbout {
  }
}
```